outputs/http.go

```diff
@@ -0,0 +1,66 @@
+package outputs
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/aquasecurity/postee/layout"
+)
+
+type HTTPClient struct {
```

Comment:
@simar7 
there is a webhook output that is doing something similar. Probably we should combine required features.

Reply:
Yes but they serve different purposes. 

Webhook sends the Postee event to the external service. While as HTTP action can technically send anything. It also includes the Postee event payload as a header.